distutils/tests/unix_compat.py

```diff
@@ -1,10 +1,15 @@
+from __future__ import annotations
+
 import sys
+from types import ModuleType
 
+grp: ModuleType | None = None
+pwd: ModuleType | None = None
 try:
     import grp
     import pwd
 except ImportError:
-    grp = pwd = None
+    pass
```

Comment:
I don't love the asymmetry of this syntax, but if this is the new Pythonic style, I guess it's acceptable. Is it?
It's starting to feel like we need a meta language to express Python imports. It would be so much nicer if we could simply express:
```
opt_import grp
opt_import pwd
```

Reply:
It's possible that this can be solved using mypy 1.18's `allow-redefinition`